src/ast/mod.rs

```diff
@@ -328,6 +328,37 @@ impl fmt::Display for DictionaryField {
     }
 }
 
+/// Represents a Map expression.
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct Map {
+    pub fields: Vec<MapField>,
```

Comment:
```suggestion
    pub entries: Vec<MapEntry>,
```
we can probably use entry since field could be confused for the key part of the map?

Reply:
Thanks @iffyio. Sounds great!